crates/astria-composer/local.env.example

```diff
@@ -14,3 +14,11 @@ ASTRIA_COMPOSER_ROLLUPS="astriachain::ws://127.0.0.1:8545"
 # Private key for the sequencer account used for signing transactions
 # Must be a hex-encoded 32-byte array (64-character hex string)
 ASTRIA_COMPOSER_PRIVATE_KEY="2bd806c97f0e00af1a1fc3328fa763a9269723c8db8fac4f93af71db186d6e90"
+
+# Block time in milliseconds
+# Should match the sequencer node configuration for 'timeout_commit', as specified in https://docs.tendermint.com/v0.34/tendermint-core/configuration.html
+ASTRIA_COMPOSER_BLOCK_TIME=2000
```

Comment:
```suggestion
ASTRIA_COMPOSER_BLOCK_TIME_MS
```
Better to make it clear that this is milliseconds because you won't see the comment in a k8s configmap